cpp/QuadratureRule.h

```diff
@@ -71,17 +78,19 @@ class QuadratureRule
   std::span<const double> weights(int i) const;
 
   /// Return offset for quadrature rule of the ith entity
-  const std::vector<std::size_t>& offset() const { return _entity_offset; }
+  const std::vector<std::size_t>& offset() const;
 
 private:
+  // What is this?
   dolfinx::mesh::CellType _cell_type;
 
-  std::size_t _tdim;
-
+  // What is this?
   int _degree;
 
+  // What is this?
   basix::quadrature::type _type;
 
+  // What is this?
```

Comment:
Dimension of the sub-entities of the cell that the quadrature rule is generated for